.travis.yml

```diff
@@ -17,20 +15,14 @@ script:
   - pyflakes src
   - check-manifest || travis_terminate 1
   - python -m pytest -r sx --ignore tests/benchmarks/ --ignore tests/test_notebooks.py
-  - if [[ $TRAVIS_PYTHON_VERSION == '3.7' ]]; then black --check --diff --verbose .; fi
-after_success: if [[ $TRAVIS_PYTHON_VERSION == '3.7' ]]; then coveralls; fi
+after_success: coveralls
```

Comment:
do not use coveralls with python 2.7. This is a regression as we only ever did coveralls with python 3.7.

Reply:
@kratsg Why does this matter? The coverage is the same, no?

Reply:
We've had issues in the past of coveralls reporting different or incorrect coverage because we call it multiple times during the CI (for different python versions). We should stick with 3.7 since we're quietly dropping 2.7 anyway.

Reply:
> We've had issues in the past of coveralls reporting different or incorrect coverage because we call it multiple times during the CI (for different python versions). We should stick with 3.7 since we're quietly dropping 2.7 anyway.

We're not dropping 2.7 now though, and this new configuration only runs Travis once. Are we sure it was a Python runtime version thing, and not a push event vs. pull request event?

> We should stick with 3.7

I think it doesn't make much sense to continue to test Python 3.7 more slowly on Travis just for this. I was going to work on getting Coveralls running in GitHub Actions in Python 3.7 as a separate PR, but I can work on getting it setup in this PR if you'd prefer that.

Reply:
I'd prefer that. It was a problem with coveralls being run for both python 2.7 and python 3.7 -- as well as the python 2.7 incorrectly reporting coverage consistently.. so we switched to only python 3.7 as you can see in the changes you're making.

Reply:
Okay, I'll work on getting Coveralls to GHA tomorrow.

Reply:
Done now.